src/main/kotlin/uk/gov/dwp/dataworks/services/UserValidationService.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.dwp.dataworks.services
+
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import uk.gov.dwp.dataworks.aws.AwsCommunicator
+import uk.gov.dwp.dataworks.logging.DataworksLogger
+
+@Service
+class UserValidationService {
+    companion object {
+        val logger: DataworksLogger = DataworksLogger(LoggerFactory.getLogger(UserValidationService::class.java))
+    }
+
+    @Autowired
+    private lateinit var awsCommunicator: AwsCommunicator
+    @Autowired
+    private lateinit var jwtParsingService: JwtParsingService
+
+    fun checkAttributes(jwt: String): Boolean {
```

Comment:
Better name would be checkJwtFor....   similar to other method name in this class.